test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -26,13 +26,19 @@ const loopbackSuccesskMock = RequestMock()
     }
   })
   .onRequestTo(/2000|6511\/probe/)
-  .respond(null, 500, { 'access-control-allow-origin': '*' })
+  .respond(null, 500, { 
+    'access-control-allow-origin': '*',
+    'access-control-allow-headers': 'X-Okta-Xsrftoken, Content-Type'
```

Comment:
We add this header because otherwise our test doesn't get past the pre-flight request.